OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseUnitWebViewFragment.java

```diff
@@ -26,6 +27,9 @@
     @InjectView(R.id.auth_webview)
     private AuthenticatedWebView authWebView;
 
+    @InjectView(R.id.swipe_container)
+    protected SwipeRefreshLayout swipeContainer;
```

Comment:
Can't we disable the swipe refresh layout in the xml ?

Reply:
No farhan we can't disable in xml